zh-learn-infrastructure/src/main/java/com/zhlearn/infrastructure/anki/AnkiNoteParser.java

```diff
@@ -0,0 +1,95 @@
+package com.zhlearn.infrastructure.anki;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Parser for the Anki collection TSV export (Chinese.txt).
+ * Only rows with note type "Chinese" or "Chinese 2" are returned.
+ */
+public class AnkiNoteParser {
+
+    private static final CSVFormat TSV = CSVFormat.DEFAULT
+        .builder()
+        .setDelimiter('\t')
+        .setQuote('"')
+        .setIgnoreEmptyLines(false)
+        .setTrim(false)
+        .build();
+
+    public List<AnkiNote> parseFile(Path file) throws IOException {
+        try (Reader r = Files.newBufferedReader(file, StandardCharsets.UTF_8)) {
+            return parseFromReader(r);
+        }
+    }
+
+    public List<AnkiNote> parseFromReader(Reader reader) throws IOException {
+        List<AnkiNote> notes = new ArrayList<>();
+        try (CSVParser parser = TSV.parse(reader)) {
+            for (CSVRecord record : parser) {
+                if (record.size() == 0) continue;
+                String first = record.get(0);
+                if (shouldSkipLine(first)) continue;
+
+                String noteType = first != null ? first.trim() : "";
+                if (!isChineseType(noteType)) continue;
+
+                String col1 = get(record, 1);
+                String col2 = get(record, 2);
+                String pinyin;
+                String simplified;
+                // Column order depends on note type
+                if ("Chinese".equals(noteType)) {
```

Comment:
I updated my collection and removed all notes of type "Chinese", so we can simplify here